src/utils/polyfill.ts

```diff
@@ -0,0 +1,19 @@
+// Add support for SVGElement.contains in IE11
+if (!SVGElement.prototype.contains) {
```

Comment:
Could probably move this to the /src folder next to BoxAnnotations. It's not really a utility module.